patient-app/README.md

```diff
@@ -1,30 +1,45 @@
-# CovidResponseApp
+# Covid Response Patient App
 
 Covid Response App is an open-source self-assessment COVID-19 tool. This repository includes code for the app to run on local machine.
 
-## App URL
-
-* [COVID Patient App](https://covid19.smilecdr.com/covid-response-app/)
+**Demo**:  [Patient App Demo](https://covid19.smilecdr.com/covid-response-app/) 
  
-## Prerequisites
+# Prerequisites
 
-* [Node](https://nodejs.org/en/) 
-* NPM :- Packaged with Node
+* [Node](https://nodejs.org/en/) and [NPM](https://covid19.smilecdr.com/covid-response-app/) (packaged with Node)
 * [Angular CLI](https://cli.angular.io/)
 
-## Local Setup
+# Installing 
+
+Clone the repository
+```
+git clone https://github.com/hapifhir/covid-response-app.git
+```
+Change to the patient-app directory
```

Comment:
It will be necessary for a user to know that the `patient-app` directory will be in the root of cloned repo.